From 6e9b9b8a457ce46858e300334b2d2eb83ac43f04 Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.j.ledkov@intel.com>
Date: Fri, 20 Feb 2015 11:27:19 +0000
Subject: [PATCH] Fix test case.
Organization: Intel Corporation (UK) Ltd. - Co. Reg. #1134945 - Pipers Way, Swindon SN3 1RJ

Under mock, mockbuild can write to /. Use /root instead.
---
 trust/test-token.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: p11-kit-0.22.1/trust/test-token.c
===================================================================
--- p11-kit-0.22.1.orig/trust/test-token.c	2015-02-20 11:48:50.362789275 +0000
+++ p11-kit-0.22.1/trust/test-token.c	2015-02-20 11:49:42.274790815 +0000
@@ -241,7 +241,7 @@
 #ifdef OS_UNIX
 	if (getuid () != 0) {
 #endif
-		token = p11_token_new (333, "/", "Label");
+		token = p11_token_new (333, "/root/", "Label");
 		assert (!p11_token_is_writable (token));
 		p11_token_free (token);
 #ifdef OS_UNIX
@@ -252,7 +252,7 @@
 	assert (!p11_token_is_writable (token));
 	p11_token_free (token);
 
-	token = p11_token_new (333, "/non-existant", "Label");
+	token = p11_token_new (333, "/etc/non-existant/", "Label");
 	assert (!p11_token_is_writable (token));
 	p11_token_free (token);
 }
